lib/oxidized/model/stoneos.rb

```diff
@@ -0,0 +1,38 @@
+class Stoneos < Oxidized::Model
+#Hillstone Networks StoneOS software
+
+  prompt /^\r?[\w.()-]+[#>](\s)?$/
+  comment '# '
+  
+  expect /^\s.*--More--.*$/ do |data, re|
+   send ' '
+   data.sub re, ''
+  end
+ 
+  cmd :all do |cfg|
+    cfg = cfg.lines.to_a[1..-2].join
```

Comment:
There should be no need for assignment here, simply doing `cfg.lines.to_a[1..-2].join` should suffice.

Reply:
`each_line` rather than `lines` as the latter has been deprecated.